docs/source/deploying-volttron/proxy_web_server.rst

```diff
@@ -0,0 +1,277 @@
+for the apache2 web server. The default location is at /etc/apache2/apache2.conf.
+
+An example apache2.conf config file can be found at volttron/scripts/admin/apache-proxy/apache2.conf.
+Within the file, you will need to set the User and Group options, with ServerAdmin being optional.
+
+.. code-block:: console
+
+    ServerRoot "/etc/apache2"
+
+    # Set User, Group, and ServerAdmin based on your system and organization
+
+    # User will be a unix user
+    User volttron
+
+    # Group will be a unix group
+    Group volttron
+
+    # ServerAdmin will be an email address.
+    ServerAdmin admin@volttron
+
+    <Directory />
+        AllowOverride none
+        Require all denied
+    </Directory>
+    <IfModule dir_module>
+        DirectoryIndex index.html
+    </IfModule>
+    <Files ".ht*">
+        Require all denied
+    </Files>
+    ErrorLog "logs/error_log"
+    LogLevel warn
+    <IfModule log_config_module>
+        LogFormat "%h %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-Agent}i\"" combined
+        LogFormat "%h %l %u %t \"%r\" %>s %b" common
+        <IfModule logio_module>
+          LogFormat "%h %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-Agent}i\" %I %O" combinedio
+        </IfModule>
+        CustomLog "logs/access_log" combined
+    </IfModule>
+    <IfModule alias_module>
+        ScriptAlias /cgi-bin/ "/var/www/cgi-bin/"
+    </IfModule>
+    <Directory "/var/www/cgi-bin">
+        AllowOverride None
+        Options None
+        Require all granted
+    </Directory>
+    <IfModule mime_module>
+        TypesConfig /etc/mime.types
+        AddType application/x-compress .Z
+        AddType application/x-gzip .gz .tgz
+        AddType text/html .shtml
+        AddOutputFilter INCLUDES .shtml
+    </IfModule>
+    AddDefaultCharset UTF-8
+    <IfModule mime_magic_module>
+        MIMEMagicFile conf/magic
+    </IfModule>
+    EnableSendfile on
+
+    # Set Security headers
+    ServerTokens Prod
+    TraceEnable Off
+    Header always set X-Frame-Options "SAMEORIGIN"
+    Header always set X-Xss-Protection "1; mode=block"
+    Header always set X-Content-Type-Options "nosniff"
+    Header always set X-Permitted-Cross-Domain-Policies "none"
+    Header unset ETag
+    FileETag None
+    Header Unset X-Powered-By
+
+    # Force redirect of http to https
+    RewriteEngine On
+    RewriteCond %{HTTPS} off
+    RewriteRule ^/+(.*) https://%{HTTP_HOST}%{REQUEST_URI} [R=permanent,L]
+
+    # Include module configuration:
+    IncludeOptional mods-enabled/*.load
+    IncludeOptional mods-enabled/*.conf
+
+    # Include list of ports to listen on
+    Include ports.conf
+
+    # Include generic snippets of statements
+    IncludeOptional conf-enabled/*.conf
+
+    # Include the virtual host configurations:
+    IncludeOptional sites-enabled/*.conf
+
+
+000-default.conf File
++++++++++++++++++++++
+
+The 000-default.conf contains the site specific configuration to serve the VOLTTRON web server from and enforce https.
```

Comment:
might include that the 000 is a sort ordering mechanism so this is the default site for a multi-site system.

Reply:
it's proxying the volttron web server not serving directly.

Reply:
Updated for clarity.